package/utils/px5g-wolfssl/px5g-wolfssl.c

```diff
@@ -144,7 +144,7 @@ int selfsigned(WC_RNG *rng, char **arg) {
   while (*arg && **arg == '-') {
     if (!strncmp(*arg, "-der", 4)) {
       pem = false;
-    } else if (!strncmp(*arg, "-newkey", 6) && arg[1]) {
+    } else if (!strncmp(*arg, "-newkey", 7) && arg[1]) {
```

Comment:
You should better check even for the terminating NULL byte. In this `strncmp()` you should check for 8 bytes. Even with your patch this would match `-newkeyfoo` for example.

Reply:
Yes. I replaced `strncmp()` like this one with `strcmp()` in the update.